app/build.gradle

```diff
@@ -168,6 +168,7 @@ dependencies {
     implementation 'net.yslibrary.keyboardvisibilityevent:keyboardvisibilityevent:2.1.0'
     implementation 'com.airbnb.android:lottie:2.2.5'
     implementation 'com.firebase:firebase-jobdispatcher:0.8.2'
+    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
```

Comment:
why is this added if you aren't using it?

Reply:
Studio must've included this when I added a new Activity, will remove it.